### DISCUSSION TITLE
Tap maintenance and Homebrew development

### DISCUSSION THREAD

**Discussion by bliu13** [Homebrew/discussions#5522] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.10
ORIGIN: https://github.com/Homebrew/brew
HEAD: c3ae0327c01aab461531bfc0fcdfb4dd3a779205
Last commit: 8 days ago
Core tap JSON: 26 Jul 15:50 UTC
Core cask tap JSON: 26 Jul 15:50 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 12
Homebrew Ruby: 3.3.3 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.3/bin/ruby
CPU: dodeca-core 64-bit arm_lobos
Clang: 15.0.0 build 1500
Git: 2.45.2 => /opt/homebrew/bin/git
Curl: 8.6.0 => /usr/bin/curl
macOS: 14.5-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
% brew doctor
Your system is ready to brew.
```

### Description of issue

[Sauce Labs Proxy Connect 5](https://docs.saucelabs.com/secure-connections/sauce-connect-5/) is out. The [cask](https://github.com/Homebrew/homebrew-cask/blob/12f2cb694fbb50b656732506ebe5f04ad971cecd/Casks/s/sauce-connect.rb) is still on version 4. What do we need to do to get this cask updated?

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Updating-Software-in-Homebrew

**Comment by daeho-ro** [comment]
The new version is supplied from their own tap, so the cask does not need to be updated, I think.

https://github.com/saucelabs/homebrew-tap

**Reply by p-linnane** [reply]
We do not remove software from Homebrew due to upstream having their own tap.

**Reply by daeho-ro** [reply]
I don't mean to remove but there was a cli changes from version 4 to 5, that may cause the version compatible problem.

**Reply by bliu13** [reply]
There's definitely breakages in the cli. You'll need to create a yaml configuration instead of passing in arguments. Your yaml file may look something like this:

```
username: some-username
access-key: some-access-key
region: some-region
tunnel-name: some-tunnel-name
```

**Comment by p-linnane** [comment, marked as answer]
I've opened https://github.com/Homebrew/homebrew-cask/pull/180782 to update to the latest version.
